desktop/lib.go

```diff
@@ -159,24 +188,48 @@ func setSelectTab(ttab *C.char) {
 
 //export plans
 func plans() *C.char {
-	resp, err := proClient.Plans(userConfig())
```

Comment:
I'm no longer seeing the correct plans when I force country to something else. Instead of doing this via settings can we cache plans using the HTTP client instead? That way if the country changes, then we will actually fetch/display the correct plans

Reply:
Same thing happens when I change my language to something else (the plans are not updated)

Reply:
@jigar-f Maybe we look into merging https://github.com/getlantern/lantern-client/pull/1033 first and combine it with the changes here to cache plans? I definitely think we should combine it with the change to fetch plans in the background ahead of time

Reply:
Sure @atavism, I am good with that, Let work on merging that first.

Reply:
Thanks, @jigar-f!